doc/adr/template.md

```diff
@@ -0,0 +1,72 @@
+# [short title of solved problem and solution]
+
+* Status: [proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)] <!-- optional -->
+* Deciders: [list everyone involved in the decision] <!-- optional -->
+* Date: [YYYY-MM-DD when the decision was last updated] <!-- optional -->
```

Comment:
Should be make those things required?

Reply:
I think the date field is redundant for a file under source control

Reply:
True

Reply:
I think it would be good if there would be a quick link to the PR though

Reply:
But the link to the PR will be available through the eventual squashed commit and then in the PR you can see who drove the decision.